bootstrap.py

```diff
@@ -232,6 +236,11 @@ def register_systems(org_name, activationkey, release):
 def unregister_system():
     print_generic("Unregistering")
     exec_failexit("/usr/sbin/subscription-manager unregister")
+    
+    
+def enable_satellite_tools():
```

Comment:
that's a downstream Satellite thing, now replaced with `--enablerepos` upstream.